apex-squid/src/test/resources/metrics/comments.cls

```diff
@@ -0,0 +1,20 @@
+/*
+    Some license for this class 
+    * comment with star
+    this license uses "double quotes"
+ 
+ */
+/* another comment*/
+/**/
+//NOSONAR
```

Comment:
I think this code will break the plugin, once we implement the ability to recover from parser errors   this shouldn't be a problem.